### PR TITLE
Adding  DIVD-2024-00051

### PR DESCRIPTION
**Pull request by flor1der**
Adding  DIVD-2024-00051

### CODE REVIEW

File: _cases/2024/DIVD-2024-00051.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+layout: case
+title: "Improper authorization vulnerabilty in ProjectSend"
+author: Florian Krijt
+lead: Koen Schagen
+excerpt: "Improper authorization vulnerabilty, cve-2024-11680, in open-source file-sharing application: ProjectSend,"
```
Comment (oscar-vlugt) [inline _cases/2024/DIVD-2024-00051.md:6]: Trailing komma after ProjectSend

File: _cases/2024/DIVD-2024-00051.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+researchers:
+- Florian Krijt
+- Koen Schagen
+cves:
+- CVE-2024-11680
+product:
+- ProjectSend
+versions: 
+- ealier then r1720
+recommendation: "Upgrade to R1720 or later"
```
Comment (oscar-vlugt) [inline _cases/2024/DIVD-2024-00051.md:16]: I prefer either the capital R or the lowercase R throughout the entire case file to keep the style the same.

File: _cases/2024/DIVD-2024-00051.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+## Recommendations
+
+To remediate {% cve CVE-2024-11680 %}, upgrade ProjectSend to version r1720 or later to resolve the improper authorisation vulnerability. Limit public access by applying strict network controls and review server logs for unusual activity, especially targeting `options.php` or unauthorised uploads in `upload/files/`. For any compromised systems, remove malicious files, restore original configurations, and investigate further for signs of exploitation. Establish a patch management process to ensure timely updates and minimise exposure to future vulnerabilities
+
+## What we are doing
+
+DIVD is currently working to identify parties that are running a vulnerable version of ProjectSend and to notify these parties. 
+
+{% include timeline.html %}
```
Comment (oscar-vlugt) [inline _cases/2024/DIVD-2024-00051.md:45]: We should add a section that is labeled "More Information" after the timeline that has some information on the CVEs, the potential exploit and maybe a blog/write-up for reference when that is available. Refer to other case files to find an example layout.

**Review by oscar-vlugt — CHANGES_REQUESTED** [review]
Looks great overall. Some remarks hat should be easy to tackle.

File: _cases/2024/DIVD-2024-00051.md (hunk 2 above)
Comment (flor1der) [reply _cases/2024/DIVD-2024-00051.md]: Adjusting everything to r1720 since that's the official naming convention used as per the ProjectSend website.

**Comment by flor1der** [comment]
Thanks @oscar-vlugt, the requested changes have been made and the pull request is ready for re-review.

**Review by oscar-vlugt — APPROVED** [review]
LGTM

**Pull request merged**